code/modules/wiremod/components/ntnet/ntnet_request.dm

```diff
@@ -0,0 +1,35 @@
+/**
+ * # NTNet Request Component
+ *
+ * Sends a data package through NTNet
+ */
+
+/obj/item/circuit_component/ntnet_send
+	display_name = "NTNet Request"
```

Comment:
"Request" is essentially a synonym for "receive", "NTNet Transmitter" might be better instead.